client/src/components/DataFiles/DataFilesListing/DataFilesListingCells.jsx

```diff
@@ -129,22 +133,44 @@ LastModifiedCell.propTypes = {
   cell: PropTypes.shape({ value: PropTypes.string }).isRequired,
 };
 
-export const FileIcon = ({ format }) => {
+export const FileIcon = ({ format, path }) => {
   const isFolder = format === 'folder';
-  const iconName = isFolder ? 'folder' : 'file';
-  const iconLabel = isFolder ? 'Folder' : 'File';
-
+  const isTrash =
+    path === '/' + useSelector((state) => state.workbench.config.trashPath);
+  let iconName = 'file';
+  let iconLabel = 'File';
+  if (isFolder) {
+    iconName = 'folder';
+    iconLabel = 'Folder';
+    if (isTrash) {
+      iconName = 'trash';
+      iconLabel = 'Trash';
+    }
+  }
   return <Icon name={iconName}>{iconLabel}</Icon>;
 };
 FileIcon.propTypes = {
   format: PropTypes.string.isRequired,
+  path: PropTypes.string,
+};
+FileIcon.defaultProps = {
+  path: '',
 };
 
 export const FileIconCell = ({ cell }) => {
-  return <FileIcon format={cell.value} />;
+  return (
```

Comment:
@rstijerina I added another var here for FileIcon to read, and to keep the accessor simple, I am bringing in the whole row, and then pick out what I need from there. I made the change in SiteSearchListing to accommodate this so errors do not come up in the future (this tries to read two vars from cell.original.row.format alone).